comixed-web/src/assets/i18n/fr/library.json

```diff
@@ -154,6 +154,7 @@
     "tooltip": {
       "deselect-all": "Déselectionner toutes les Bandes Dessinées.",
       "navigation-tree": "Affichier la bibliothèque en vue arborescente",
+      "scrape-comics": "Commencer á recuperation des details",
```

Comment:
"Commencer à récupérer les détails" would be more correct. :)
A better translation would be : "Commencer la récupération des détails".

Reply:
Thank you! Yes, "les" vs. "des" makes more sense since I'm getting specific data for a specific issue. Merci.